jax/_src/numpy/array_methods.py

```diff
@@ -909,15 +909,15 @@ def max(self, values, *, indices_are_sorted=False, unique_indices=False,
   "setitem": _unimplemented_setitem,
   "copy": _copy,
   "deepcopy": _deepcopy,
-  "neg": ufuncs.negative,
-  "pos": ufuncs.positive,
+  "neg": lambda self: ufuncs.negative(self),
```

Comment:
These need an extra wrapper because when a class attribute is set ot a class instance, Python assumes it's a descriptor, but we want it to be a method.